mod/deckorate_core/set/type/metric/export.rb

```diff
@@ -108,3 +108,65 @@ def semicolon_separated_values column
     card.send("#{column}_card").item_names.join ";"
   end
 end
+
+format :jsonld do
+  
+  def molecule
+    metric_jsonld(atom)
+  end
+
+  private
+
+    def metric_jsonld(a)
+    {
+      "@context" => "#{request.base_url}/context/#{card.type}.jsonld",
```

Comment:
You can make a method for the context url in an abstract set and reuse it. It's the same everywhere